src/autogpt_plugins/email/email_plugin/email_plugin.py

```diff
@@ -311,3 +312,58 @@ def clean_email_body(email_body):
     email_body = re.sub(r"http\S+", "", email_body)
 
     return email_body
+
+def truncate_emails(messages):
+    """Truncate email content based on configuration settings
+
+    Args:
+        messages (messages, optional): List of emails
+
+    Returns:
+        messages: List of truncated emails
+    """
+
+    # If MAX_CHARACTERS is breached
+    messages_length = len(str(messages))
+    if os.getenv("MAX_CHARACTERS") and int(os.getenv("MAX_CHARACTERS")) < int(messages_length):
```

Comment:
You should also add a test, when these env vars are not set. In this case we must use default values.